:class:`trio.testing.RaisesGroup` can now catch an unwrapped exception with ``unwrapped=True``, similar to ``except*``.
